gremlin-go/driver/graph.go

```diff
@@ -74,7 +74,7 @@ type Property struct {
 // A list of labels and a list of objects is maintained in the path.
 // The list of labels are the labels of the steps traversed, and the objects are the objects that are traversed.
 type Path struct {
```

Comment:
```suggestion
// TODO change labels to be []<set of string> after implementing set in AN-1022 and update the GetPathObject accordingly
type Path struct {
```
Adding a TODO here as a reminder